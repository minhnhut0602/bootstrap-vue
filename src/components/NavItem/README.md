# NavItem

```vue live
<NavItem>Default Example Usage</NavItem>
```

## Slots

| Name    | Description | Bindings |
| ------- | ----------- | -------- |
| default |             |          |
