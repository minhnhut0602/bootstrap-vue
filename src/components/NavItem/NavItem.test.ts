import NavItem from '.';

describe('template', () => {
    componentRenderTest(NavItem);

    componentSlotRenderTest(NavItem);
});
