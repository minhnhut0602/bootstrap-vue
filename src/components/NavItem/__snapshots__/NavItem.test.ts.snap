// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template renders default 1`] = `
"<li class=\\"nav-item\\">
  <!-- @slot Displays the content of the nav item -->
</li>"
`;
